bigquery_etl/parse_udf.py

```diff
@@ -34,32 +34,54 @@ class RawUdf:
     def from_file(filepath):
         """Read in a RawUdf from a SQL file on disk."""
         dirpath, basename = os.path.split(filepath)
+
+        with open(filepath) as f:
+            text = f.read()
+
+        sql = sqlparse.format(text, strip_comments=True)
+        statements = [s for s in sqlparse.split(sql) if s.strip()]
+
         prod_name = basename.replace(".sql", "")
-        internal_name = os.path.basename(dirpath) + "_" + prod_name
+        persistent_name = os.path.basename(dirpath) + "." + prod_name
+        temp_name = os.path.basename(dirpath) + "_" + prod_name
+        internal_name = None
```

Comment:
This is to both support temporary and persistent UDFs. (Temporary UDFs, such as `assert_false`, are for example still used for tests)